mumd/src/audio.rs

```diff
@@ -354,3 +381,21 @@ impl Audio {
         play_sounds.extend(samples.iter().skip(l));
     }
 }
+
+// moo
+fn get_sfx(file: &str) -> Cow<'static, [u8]> {
+    let mut buf: Vec<u8> = Vec::new();
+    if let Ok(mut file) = File::open(file)
+        .or_else(|_| File::open(format!("/home/gustav/dev/mum/mumd/src/resources/{}", file)))
```

Comment:
This path shouldn't be `/home/gustav/...`

Reply:
Indeed